README.md

```diff
@@ -1,25 +1,75 @@
-# New Project
+# Welcome to Doenet!
 
-> ✨ Bootstrapped with Create Snowpack App (CSA).
+The Distributed Open Education Network (Doenet) is an open data-driven educational technology platform designed to measure and share student interactions with web pages. It includes tools for authoring interactive educational content, including our custom DoenetML markup language, and conducting educational research using the content. Our ultimate goal is to provide research-based tools to help instructors and learners discover the most effective content. Simply put, Doenet gives teachers complete flexibility over their educational content and gives them power of anonymized student data to track learning outcomes. With Doenet, we hope to help teachers teach better and students learn better.
 
-## Available Scripts
+Although we are still in the early stages, we are excited to introduce Doenet and illustrate the richly interactive activities that one can author with it.
 
-### npm start
+For more background and information on the Doenet project, see this [MAA DUE Point](https://www.mathvalues.org/masterblog/reimagining-online-mathematics) article.
 
-Runs the app in the development mode.
-Open http://localhost:8080 to view it in the browser.
+We would love to hear from you! Join our [Discord](https://discord.gg/PUduwtKJ5h) to ask questions and stay updated on our progress!
+
+## What We Use
+
+Doenet is built using React and bundled for the web using Snowpack. Data for Doenet is stored in MySQL tables and components have been tested in Cypress. We use Docker to ship and run Doenet and our container uses Apache, Snowpack, PHP, MySQL, and RTNode. For reference, the container is set up to use port 80 for the local Downet website, port 8080 for the build site, and port 3306 for MySQL. You may have to change the ports used on your local device.
+
+<!-- Need build instructions from Kevin  -->
+
+## Getting Started
+
+Doenet is open-source so that instructors and learners can customize it to best serve their needs. To start developing:
+
+1. Fork the DoenetTools repository.
+
+   To enable the v2 docker compose CLI on Linux, [install the docker-compose-plugin](https://docs.docker.com/compose/install/linux/).
+
+2. Open the repository and run `npm install` to install all modules used by Doenet.
+
+3. To start your local version of Doenet, run `npm start` and view it in the browser at http://localhost:80.
+
+- If you are working with databases, we recommend installing [Sequel Ace](https://sequel-ace.com/) on MacOS.
+<!-- Any recs for Windows/Linux? -->
```

Comment:
@dqnykamp You use MySQL workbench right?

Reply:
That's right.